### ISSUE TITLE
Can't move widgets in dashboards

### ISSUE THREAD

**Issue by battosai30** [GladysAssistant/Gladys#1967]
Hi,

**Describe the bug**
Can't move widgets in dashboards

**To Reproduce**

1. Edit dashboard
2. Add a widget
3. Try to move it :) => fail
4. Save dashboard
5. Click Edit
6. Try to move it :) => still fail

**Expected behavior**

Being able to move it :D 

**Screenshots**
![image](https://github.com/GladysAssistant/Gladys/assets/5184702/f3c881af-e20f-4ac7-949b-20ab85cbb3cd)

**Your Gladys installation (please complete the following information):**

Linux Debian 12 in LXC container (Host = Proxmox V8.1)
Gladys v4.31.1

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser : tried on Firefox, Edge and Chrome

Thank for your work !

**Comment by Pierre-Gilles** [comment]
@battosai30 Thanks for the bug report! Do you have a touch screen on your laptop? The library we currently use for drag and drop support either touch or mouse support, but not both. So if you have a touch-screen, it overrides mouse support... :D

**Comment by battosai30** [comment]
No it's a no-touch screen. But on my ipad it works with touch. So I guess we are not so far from the issue's source ...
I disconnected everything from my laptop, I still can't drag widgets :(
I took a look at my devices manger, disabled some touch virtual stuff => fail. But I guess it's with kind of thing.
I think we can close it as I may be the only one concerned (never formated my PC for years). If someone meet the same issue we may have some more informations
